Framework/Foundation/include/Framework/Pack.h

```diff
@@ -272,6 +272,29 @@ constexpr auto unique_pack(pack<T, Ts...>, PT p2)
 template <typename P>
 using unique_pack_t = decltype(unique_pack(P{}, pack<>{}));
 
+template <typename P, typename Is>
+struct pack_to_tuple;
+
+template <typename P, std::size_t... Is>
+struct pack_to_tuple<P, std::index_sequence<Is...>> {
+  using type = std::tuple<pack_element_t<Is, P>...>;
+};
+
+template <typename P>
+using pack_to_tuple_t = typename pack_to_tuple<P, std::make_index_sequence<pack_size(P{})>>::type;
```

Comment:
Wouldn't:

```C++
template <typename...ARGS>
inline constexpr std::tuple<ARGS...> pack_to_tuple(pack<ARGS...>) { return std::tuple<ARGS...>{};}

template <typename...ARGS>
using pack_to_tuple_t = decltype(pack_to_tuple(pack<ARGS...>{}));
```

be more convenient? Or alternatively something like:

```

template <typename T>
struct pack_to_tuple_t{};

template <typename... ARGS>
struct pack_to_tuple_t<pack<ARGS...>> {
   using type = std::tuple<ARGS...>;
};
```

Sorry for being pedantic, but Pack.h is quite in the critical path for compile time performance.